dashboard/app/views/devise/registrations/edit.html.haml

```diff
@@ -15,6 +15,9 @@
   - email_mismatch = current_user.errors.delete(:email_mismatch)
   - current_user.reload if email_mismatch.present?
   = render "devise/shared/error_messages", resource: resource
+  - if current_user.permission?(UserPermission::AUTHORIZED_TEACHER)
+    .field
+      = f.label '✔ '+t('activerecord.attributes.user.verified_teacher'), class: 'label-bold', style: 'color:green; text-transform: capitalize';
```

Comment:
Usually it can be buggy to concatinate two strings together because the behaviour with RTL languages is hard to predict. Please verify this still looks good in a RTL language (Arabic / ar-SA)